### PR TITLE
test(reputation): adds compose based integration test with iprepd

### PR DESCRIPTION
**Pull request by ameihm0912**
Add framework for integration testing between fxa-customs-server and
iprepd (newer version of Tigerblood). The existing set of tests included a stub
reputation server; this conducts similar testing but uses the actual
reputation service backend.

Note: this doesn't actually hook it into the CI yet but can be executed manually

```
docker build -f Dockerfile-build fxa-customs-server:build
cd test/integration/reputation
docker-compose build
docker-compose run customs
docker-compose down
```

### CODE REVIEW

File: test/integration/reputation/iprepd.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+listen: 0.0.0.0:8080
```
Comment (vladikoff) [inline test/integration/reputation/iprepd.yaml]: @ameihm0912 is it possible to please pick a less common port?

Comment (ameihm0912) [reply test/integration/reputation/iprepd.yaml]: No problem, I'll adjust this PR

**Comment by ameihm0912** [comment]
@vladikoff this should be fixed up, and I also added a README to the test directory.

**Comment by vladikoff** [comment]
Thank you @ameihm0912 !

**Pull request merged**